README.md

```diff
@@ -30,13 +31,28 @@ Usage
 Library has the following RxJava Observables available in the public API:
 
 ```java
+Observable<Boolean> observeWifiConnection(final Context context)
 Observable<List<ScanResult>> observeWifiAccessPoints(final Context context)
 Observable<Integer> observeWifiSignalLevel(final Context context, final int numLevels)
 Observable<WifiSignalLevel> observeWifiSignalLevel(final Context context)
 Observable<SupplicantState> observeSupplicantState(final Context context)
 Observable<WifiInfo> observeWifiAccessPointChanges(final Context context)
 ```
-
+### Observing WiFi Connection
+We can observe WiFi connection with `observeWifiConnection(context)` method.
+Subscriber will be called everytime, when WiFi connection On or Off.
```

Comment:
It could be changed to `Subscriber will be called every time, when device is connected to WiFi or disconnected from WiFi.`